packages/smooth_app/lib/pages/preferences/user_preferences_settings.dart

```diff
@@ -348,3 +351,45 @@ class _CameraPlayScanSoundSetting extends StatelessWidget {
     );
   }
 }
+
+class _ExpandPanelNutritionTableSetting extends StatelessWidget {
```

Comment:
If you don't feel like creating a common class, at least please add a `TODO` mentioning that it would probably be an idea worth considering.